src/renderer/components/+workloads-overview/workloads/implementations/statefulsets-workload.injectable.ts

```diff
@@ -5,16 +5,16 @@
 import { getInjectable } from "@ogre-tools/injectable";
 import { workloadInjectionToken } from "../workload-injection-token";
 import { ResourceNames } from "../../../../utils/rbac";
-import navigateToPodsInjectable from "../../../../../common/front-end-routing/routes/cluster/workloads/deployments/navigate-to-deployments.injectable";
```

Comment:
the downside of default exports? `navigateToPodsInjectable` from `navigate-to-deployments.injectable`? 

This looks like two levels of copy-paste errors :)

Reply:
I agree with you on this. I didn't even notice the wrong name, and was thinking "how this lead to deployments.."

Reply:
But of course, there should be unit test to cover it. :) Sooo, I'd say that problem is elsewhere.